freqtrade/analyze.py

```diff
@@ -172,6 +172,10 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool, sell
         if the threshold is reached and updates the trade record.
         :return: True if trade should be sold, False otherwise
         """
+        if buy and self.config.get('experimental', {}).get('ignore_roi_if_buy_signal', False):
```

Comment:
I think `self.config.get('experimental', {}).get('ignore_roi_if_buy_signal', False)` should be made an variable to improve the readability a bit